support/fixtures/factories/contxtUserEvent.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const factory = require('rosie').Factory;
+const faker = require('faker');
+
+factory
+  .define('contxtUserEvent')
```

Comment:
This should be `contxtUserEventSubscription`

Reply:
Chatted with Greg and decided userEventSubscription is a better name for it